server/webhook_parser.go

```diff
@@ -270,7 +270,7 @@ func parseWebhookAssigned(jwh *JiraWebhook) *webhook {
 
 	// Don't send a notification to the assignee if they are the one who made the change. (They probably know already.)
 	if (jwh.User.Name != "" && jwh.User.Name == jwh.Issue.Fields.Assignee.Name) ||
-		(jwh.User.AccountID != "" && jwh.Issue.Fields.Assignee.AccountID != "" && jwh.User.AccountID == jwh.Issue.Fields.Assignee.AccountID) {
```

Comment:
removed a redundant clause

Reply:
That was to protect against matching a blank `jwh.Issue.Fields.Assignee.AccountID` with a blank `jwh.User.AccountID` -- they would both be blank if we have a jira Server instance, right?

Reply:
@levb you're right, you can remove the `jwh.Issue.Fields.Assignee.AccountID != ""` check:
```suggestion
		(jwh.User.AccountID != "" && jwh.User.AccountID == jwh.Issue.Fields.Assignee.AccountID) {
```